lively.ide/studio/dialogs.cp.js

```diff
@@ -143,20 +143,26 @@ class SaveWorldDialogModel extends ConfirmPromptModel {
   }
 }
 
-// SaveWorldDialog.openInWorld()
 const SaveWorldDialog = component(DarkPrompt, {
   defaultViewModel: SaveWorldDialogModel,
   name: 'save world dialog',
   extent: pt(470, 320),
+  layout: new VerticalLayout({
+    align: 'center',
+    autoResize: true,
+    direction: 'topToBottom',
+    renderViaCSS: true,
+    resizeSubmorphs: true,
+    spacing: 16
+  }),
```

Comment:
Can we make this a `TilingLayout`? :slightly_smiling_face: Otherwise, stuff will break later on with #547.